layer1/bridge/consensus/parameter.mligo

```diff
@@ -0,0 +1,83 @@
+module Types = struct
```

Comment:
The pattern for the Parameter.Types module is to have a `Parameter.Types.t` type to use a the parameter for the main function (replace the type `action` in your main.mligo main function
Here another exemple https://github.com/ligolang/randomness-cameligo/blob/main/contracts/parameter.mligo